docs/MO_DG/prepare_model/convert_model/Converting_Model.md

```diff
@@ -1,18 +1,16 @@
 # Setting Input Shapes {#openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model}
 
-Paragraphs below provide details about specifying input shapes for model conversion.
```

Comment:
@kblaszczak-intel could you suggest a better introduction?

Reply:
I think we could make a general intro from some content under the two headers. But we should also change the headers a bit, like "Specifying the --input_shape parameter" or even just give the name of the parameter. The article name and headers don't give the idea of what is in the article. Could we maybe work on this article as a whole in the nearest future?
For now, I would either leave it without the intro, or put something like:
Model Optimizer provides the option of making models more efficient by providing additional shape definition. It is achieved with two parameters: `--input_shape` and `--static_shape`, used under certain conditions.